Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitWebView.cpp

```diff
@@ -1411,6 +1412,35 @@ static void testWebViewIsWebProcessResponsive(WebViewTest* test, gconstpointer)
     g_assert_true(webkit_web_view_get_is_web_process_responsive(test->m_webView));
 }
 
+static void testWebViewIsWebProcessUnresponsive(WebViewTest* test, gconstpointer)
```

Comment:
Does the test fail if run without the rest of your changes?

Reply:
I will test that.

Reply:
If I disable creation of PeriodicalResponsivenesTimer in WebKitWebView contructor then (with additional check in webkitWebViewRestartWebProcessResponsivenessTimerForTesting) it timeouts

Reply:
Good, then you are successfully testing that your improvement works.